sound/soc/sof/mediatek/mt8186/mt8186.c

```diff
@@ -460,14 +466,59 @@ static int mt8186_get_bar_index(struct snd_sof_dev *sdev, u32 type)
 	return type;
 }
 
-static int mt8186_ipc_msg_data(struct snd_sof_dev *sdev,
-			       struct snd_pcm_substream *substream,
-			       void *p, size_t sz)
+static void *mt8186_machine_info_select(struct snd_sof_dev *sdev)
 {
-	sof_mailbox_read(sdev, sdev->dsp_box.offset, p, sz);
-	return 0;
+	struct snd_sof_pdata *sof_pdata = sdev->pdata;
+	const struct sof_dev_desc *desc = sof_pdata->desc;
+	struct snd_sof_mach_info *mach_info = (struct snd_sof_mach_info *)desc->mach_info;
+
+	for (; mach_info->board; mach_info++) {
+		int score = of_machine_is_compatible(mach_info->board);
+		dev_dbg(sdev->dev, "%s, score:%d, board:%s\n",
```

Comment:
Add one blank line between local declaration and statement. Thanks.

Reply:
done